CHANGELOG.rst

```diff
@@ -17,6 +17,21 @@ Added
 
    For backward compatibility reasons, if pack metadata file doesn't contain that attribute, it's
    assumed it only works with Python 2. (new feature) #4474
+
+Changed
+~~~~~~~
+
+* Changed the ``inquiries`` API path from ``/exp`` to ``/api/v1``. #4495
+
+Fixed
```

Comment:
Don't think we need this section if there's nothing in it?

Reply:
I plan to put an entry there right after this PR is merged.

Reply:
Ok then